src/components/pages/Categories.js

```diff
@@ -1,6 +1,17 @@
-const Categories = () => (
-  <main>
-    <button type="button">Check Status</button>
-  </main>
-);
+import { useDispatch } from 'react-redux';
+import { categoriesActions } from '../../redux/categories/categories';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  const clickHandler = () => {
+    dispatch(categoriesActions.checkStatus());
+  };
+  return (
+    <main>
+      <button type="button" onClick={clickHandler}>Check Status</button>
+    </main>
```

Comment:
- [ ] When ı click this button it is not showing 'under construction' text kindly fix i.t